plugin/web/extensions/infobox/core/components/viewer/index.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Primitive, PublicSetting } from "@web/extensions/infobox/types";
+import { Collapse } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+import { useEffect, useState } from "react";
+
+type Props = {
+  key: number;
+  primitive: Primitive;
+  publicSettings: PublicSetting[];
+};
+
+type PropertyItemType = {
+  key: string;
+  title: string;
+  value?: any;
+};
+
+const Viewer: React.FC<Props> = ({ primitive, publicSettings, key, ...props }) => {
```

Comment:
Are you expecting anything in the `...props`? Because you destructure (as far as I can tell) all your props.

Reply:
This is from Antd Collapse component, it has some difficulty in wrap Panel as a child component and this is a solution for it. Please check [here](https://github.com/react-component/collapse/issues/73#issuecomment-878312513).